meta-mel/recipes-core/os-release/os-release.bbappend

```diff
@@ -0,0 +1,13 @@
+OS_RELEASE_FIELDS = "PRETTY_NAME NAME VERSION_ID VERSION VERSION_CODENAME ID HOME_URL SUPPORT_URL BUG_REPORT_URL BUILD_ID"
+OS_RELEASE_UNQUOTED_FIELDS = "ID VERSION_ID VARIANT_ID"
+
+ID = "${DISTRO}"
+NAME = "${OS_NAME}"
```

Comment:
I'm curious, why is the name Flex OS and not Mentor Embedded Linux Flex OS, is just Flex OS the real product name? Should we use that for DISTRO_NAME?

Reply:
I think you got the answer from email.